torchci/pages/_githubrunnersutilization.tsx

```diff
@@ -0,0 +1,165 @@
+import TimeSeriesPanel from "components/metrics/panels/TimeSeriesPanel";
+import TablePanel from "components/metrics/panels/TablePanel";
+import { RocksetParam } from "lib/rockset";
+import { durationDisplay } from "components/TimeUtils";
+import { TimeRangePicker } from "./metrics";
+import { useState } from "react";
+import dayjs from "dayjs";
+import {
+  Grid,
+  Stack,
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  SelectChangeEvent,
+} from "@mui/material";
+
+const ROW_HEIGHT = 340;
+
+/**
+ * Allows the user to pick the TTS metrics.
+ */
+export function RunnerTypePicker({
+  runnerType,
+  setRunnerType,
+}: {
+  runnerType: string;
+  setRunnerType: any;
+}) {
+  function handleChange(e: SelectChangeEvent<string>) {
+    setRunnerType(e.target.value as string);
+  }
+
+  return (
+    <>
+      <FormControl>
+        <InputLabel id="github-runner-type-picker-select-label">
+          Runner Type
+        </InputLabel>
+        <Select
+          defaultValue={runnerType}
+          label="Runner Type"
+          labelId="github-runner-type-picker-select-label"
+          onChange={handleChange}
+        >
+          <MenuItem value={"macos-12"}>MacOS 12</MenuItem>
```

Comment:
Just FYI, there is a way to programmatically populate this list if you need to do so, for example https://github.com/pytorch/test-infra/blob/main/torchci/pages/benchmark/compilers.tsx#L471-L475.  The data could be hard coded here as a constant, or come from Rockset